src/DataBlobs/services/HelpersDataWrapEntryBlob.jl

```diff
@@ -51,11 +51,14 @@ BlobEntry(
     label::Symbol = entry.label, 
     blobstore::Symbol = entry.blobstore, 
     hash::String = entry.hash,
+    size::Int = entry.size,
```

Comment:
I still don't understand why you want to add the new size field to BlobEntry. I would say it lower priority and you can use meta. https://github.com/JuliaRobotics/DistributedFactorGraphs.jl/issues/985#issuecomment-1513828542

Reply:
it doesn't hurt to have this in there, and is a different use case from metadata.  Based on what I've seen the past months, Id like .size to be in please.

Reply:
I don't have a problem with .size if you want it, it's just that the API does not support it, so it will just get lost. 
Also just make sure this doesn't break the SDKs.